README.md

```diff
@@ -1,2 +1,50 @@
 # community
-Community repository for governance, issue with org, scope, meetings etc.
+
+# what is the just-foss organisation?
+
+The aim of just-foss organisation it to be a free, opensource, remote, regular meeting, written by and for hackers in computer science. 
+
+We want to provide a community which is not driven by status, but rather by content and real humanity. ( we like humor but not hyped code of conducts, ok? :) ) 
```

Comment:
I would be in favor of adopting a CoC, personally - trying to fight that is probably going to come back and bite everyone eventually. I don't think that has to be opposed to humor or humanity, hehe.

Reply:
ok done thx. PR created